### PR TITLE
First Code Review

### PR DESCRIPTION
**Pull request by Tessyrich**
Added the necessary comments on the code

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+                class="stretched"
+                type="text"
+                id="name"
+                placeholder="Enter Your Name"
+                required
+            /><br />
+            <label id="email-label">Email</label><br />
+
+            <input
+                class="stretched"
```
Comment (Tessyrich) [inline index.html:27]: Please use an indention of two lines that is [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+                placeholder="Age"
+                min="13"
+                max="200"
+            /><br />
+            <label for="">Which option best describe your current role?</label
+            ><br />
+
+            <select name="select option" id="dropdown" class="stretched">
+                <option value="Student">Student</option>
+                <option value="Full Time job">Full Time job</option>
```
Comment (Tessyrich) [inline index.html:47]: As requested in the project requirement: Please use an indentation of two lines for all your documents. This is the convention and also the [best HTML & CSS practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

File: index.html [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+                    name="recommend"
+                    value="no"
+                    id="radio-btn"
+                />
+                <label for="radio-btn">No</label>
+            </div>
+
+            <div id="checkboxes">
+                <p>What would you like to see improved?</p>
+                <input
```
Comment (Tessyrich) [inline index.html:73]: Please use an indention of two lines that is [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

File: index.html [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+                    name="improvements"
+                    id="checkbox"
+                />
+                <label for="checkbox">Backend</label>
+            </div>
+
+            <textarea
+                name="comments"
+                id="texarea"
+                cols="30"
```
Comment (Tessyrich) [inline index.html:99]: Please use an indention of two lines that is [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

File: index.html [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+        <script src="js/index.js"></script>
+    </body>
+</html>
```
Comment (Tessyrich) [inline index.html:121]: Please use an indention of two lines that is [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

File: index.html [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+
+            <select name="select option" id="dropdown" class="stretched">
+                <option value="Student">Student</option>
+                <option value="Full Time job">Full Time job</option>
+                <option value="Full timer learner">Full timer learner</option>
+                <option value="Prefer not to say">Prefer not to say</option>
+                <option value="others">others</option>
+            </select>
+
+            <div id="radio-buttons">
```
Comment (Tessyrich) [inline index.html:53]: Please use a section tag or any other tag. This is HTML & CSS best practices

**Review by Tessyrich — COMMENTED** [review]
Changes Required.### 
 You are almost there. But you need to add a little change.

Check and complete the changes below and submit them for re-review also feel free to contact me for any questions using via email, slack, or GitHub.

**Review by aimalamiri — CHANGES_REQUESTED** [review]
Hi @Tessyrich, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

AlmostThereReachGIF

## Highlights

- Good job for using the right GitHub Flow 💯
- Awesome code review comments 🗃️

## Required Changes ♻️
You did a good job! You added respectful comments, but please make sure to add some more description to your code comments.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+
+        <form id="survey-form">
+            <label id="name-label">Name</label><br />
+            <input
+                class="stretched"
+                type="text"
+                id="name"
+                placeholder="Enter Your Name"
+                required
+            /><br />
```
Comment (Tessyrich) [inline index.html:23]: As requested in the project requirement: Please use an indentation of two lines for all your documents. This is the convention and also the [best HTML & CSS practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

File: index.html (hunk 6 above)
Comment (Tessyrich) [inline index.html:53]: Kindly use a section tag or any other tag other than div to wrap a container. This best practices fro HTML see,amtics

File: index.html [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+            <div id="checkboxes">
+                <p>What would you like to see improved?</p>
+                <input
+                    type="checkbox"
+                    value="frontend"
+                    name="improvements"
+                    id="checkbox"
+                />
+                <label for="checkbox">Frontend</label>
+                <input
```
Comment (Tessyrich) [inline index.html:80]: As requested in the project requirement: Please use an indentation of two lines for all your documents. This is the convention and also the [best HTML & CSS practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

**Review by Tessyrich — COMMENTED** [review]
Required changes### 
Good job so far!
 There are minor issues you still need to work on.
Kindly check the comments under the review and submit for re-review after making changes.

**Review by aimalamiri — APPROVED** [review]
Hi @Tessyrich, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

ApprovedDavieGIF


## Highlights


- Good job for using the right GitHub Flow 💯
- Review comments are descriptive and respectful 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by andgarzonmal** [comment]
Hi @Tessyrich , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Nice job reviewing your project ✔️ 
- Good implementations  ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**